pkg/util/util_test.go

```diff
@@ -1652,3 +1652,56 @@ func TestValidateURL(t *testing.T) {
 		})
 	}
 }
+
+func TestCopyFile(t *testing.T) {
+	// Create temp dir
+	tempDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Errorf("Failed to create temp dir: %s, error: %v", tempDir, err)
+	}
+
+	// Create temp file under temp dir as source file
+	tempFile, err := ioutil.TempFile(tempDir, "")
+	if err != nil {
+		t.Errorf("Failed to create temp file: %s, error: %v", tempFile.Name(), err)
+	}
+	defer tempFile.Close()
+
+	srcPath := tempFile.Name()
+	fakePath := filepath.Join(tempDir, "fakeFile")
+	dstPath := filepath.Join(tempDir, "dstFile")
+
+	tests := []struct {
+		name    string
+		srcPath string
+		dstPath string
+		wantErr bool
+	}{
+		{
+			name:    "Case 1: Copy successfully",
+			srcPath: srcPath,
+			dstPath: dstPath,
+			wantErr: false,
+		},
+		{
+			name:    "Case 2: Source doesn't exist",
```

Comment:
I think `Case 2: Invalid source path` justifies the case concern more. no?

Reply:
That could work, done.